airbyte-config/persistence/src/main/java/io/airbyte/config/persistence/ConfigRepository.java

```diff
@@ -117,24 +117,34 @@ public void setFeedback(final UUID workflowId) throws JsonValidationException, C
     persistence.writeConfig(ConfigSchema.STANDARD_WORKSPACE, workspace.getWorkspaceId().toString(), workspace);
   }
 
-  public StandardSourceDefinition getStandardSourceDefinition(final UUID sourceDefinitionId)
+  public StandardSourceDefinition getStandardSourceDefinition(final UUID sourceDefinitionId, final boolean includeTombstone)
```

Comment:
After reading through, I don't really see any cases except for the `ConfigDumpImporter` / other cases where we are listing source definitions where we wouldn't want to return the source definition at this level whether or not it's tombstoned. 

In the future, if we're generating billing reports for custom connectors, we'd want to be able to retrieve even if the definition was deleted. This kind of guard against including tombstones makes the consumer of everything that uses it need to consider whether or not tombstoning is happening.

I think there are two options really:
1. make this function always return the value (and remove the `includeTombstone` parameter) and move all of the actual filtering to `ConfigDumpImporter` / listing use cases.
2. change all of the `false` versions of this to `true` except for the `ConfigDumpImporter` / listing use cases.

Reply:
@jrhizor i'm not sure i fully understood what you were recommending. lmk if what i'm saying is compatible.

* `get` whenever we get an object, i think we want to get it regardless of whether it is tombstoned or not. so i don't think we need the boolean for the `getStandardSourceDefinition` or `getSourceDefinitionFromSource` or `getSourceDefinitionFromConnection`
* `list` i do think for list we want to be able to choose. e.g. `listStandardSourceDefinitions`.

Reply:
That's basically option 1. 

Do we need anything special for the `ConfigDumpImporter` usage in that case?

Reply:
Actually Parker already added the special case in the importer, so looks good to me.